easy_nodes/easy_nodes.py

```diff
@@ -712,6 +715,8 @@ def ComfyNode(
     debug: bool = False,
     color: str = None,
     bg_color: str = None,
+    height: int = None,

```

Comment:
Can you reverse the order please? Typically width comes before height in parameter lists

Reply:
Order switched in most recent commit (74f8f68)